data/strings/types_strings.txt

```diff
@@ -9011,8 +9035,8 @@
     tr = Anıt
     uk = Дошка пам'яті
     vi = Đài tưởng niệm
-    zh-Hans = 纪念物
-    zh-Hant = 紀念館
+    zh-Hans = 纪念牌匾
+    zh-Hant = 紀念牌匾 
```

Comment:
```suggestion
    zh-Hant = 紀念牌匾
```

Reply:
extra whitespace will be stripped automatically by `generate_localizations.sh`, so its not an issue